addons/web/static/src/search/search_bar/search_bar.js

```diff
@@ -70,6 +71,15 @@ export class SearchBar extends Component {
 
         const tasks = [];
         for (const id of expanded) {
+            // expand a properties should lazily fetch the definition, and generate
+            // searchItems, so the normal field logic is used for properties
+            const searchItem = this.searchItems.find((i) => i.id === id);
+            const field = this.fields[searchItem.fieldName];
+            if (field.type === "properties" && searchItem.type !== "field_property") {
+                await this._getPropertiesSearchItems(searchItem, field, query);
+                continue
```

Comment:
Unnecessary continue?
(Plus missing semicolon, make sure to check that linter :eyes: )